@plotly/dash-test-components/src/components/CollapseComponent.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
```

Comment:
Adding a set of components with simple behaviors corresponding to various usage scenarios. This component hides its content by default (simplified version of what dcc.Tabs/dcc.Tab do)